pkgs/servers/sql/postgresql/libpq.nix

```diff
@@ -12,6 +12,7 @@
   # build dependencies
   bison,
   flex,
+  makeWrapper,
```

Comment:
How about using `makeBinaryWrapper` instead?

Reply:
I just would simply use `makeBinaryWrapper` by default everywhere, since if offers better compatibility with Darwin systems. That's the only reason for me.

Reply:
Fair enough. My comment was not a blocker so, feel free to implement what you prefer.